lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -70,6 +72,8 @@ def test_bold_font_output():
     ax.set_title('bold-title', fontweight='bold')
 
 
+@pytest.mark.skipif(sys.platform == 'darwin',
+                    reason="Fails on darwin")
```

Comment:
This is related to the svg to png conversion. The resulting svg file is identical to a local test, but the png is slightly different.

Reply:
This is the diff:
![image](https://user-images.githubusercontent.com/8114497/183970343-657d8700-3510-4625-843d-7629be34e039.png)

Reply:
I think this will be fixed by one of the last commits in #23559 which adds quotes to the font name in the expected svg.

I think they were always mal-formed (missing quotes on the family name), but inkscape (probably actually librsvg) is getting crankier about it.  At some point we fixed the bug of not putting in quotes so the rest image is correct and the expected image is wrong 🤦🏻 .

Reply:
> I think this will be fixed by one of the last commits in #23559 which adds quotes to the font name in the expected svg.

Seems like that didn't help. Should I simply go back to skipping (or possible better xfail-ing if that can be done on a platform-specific basis)?

Reply:
xfail for now and I'll look into if I can reproduce this on a local mac.